Topotify/Views/TabBarViews/TopArtistsView.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  TopArtistsView.swift
+//  Topotify
+//
+//  Created by Usama Fouad on 23/07/2022.
+//
+
+import SwiftUI
+import Combine
+import SpotifyWebAPI
+import SpotifyExampleContent
+
+
+struct TopArtistsView: View {
+    
+    @EnvironmentObject var spotify: Spotify
+
+    @State private var topArtists: [Artist]
```

Comment:
Just set the initial value at the declaration site and get rid of the initializer.